developer_workflow.md

```diff
@@ -1,41 +1,87 @@
+
+# Card Review
+
+During the planning meeting, the team will discuss with the client each item (which we refer to as a card) to be worked on.  Before we can proceed with any development we need to review each card, covering the card review elements below.  Our goal is to keep as much discussion as possible in the planning meeting, trying to avoid the need to have additional meetings, email exchanges or chat in order for a developer to start and complete work on the card.  Not every card will need this much discussion but consider each of these items and determine if it's appropriate to bring into the discussion.  Again, we don't want to waste time but we also want to avoid misunderstandings or miscommunication about what we need to do on the card.
+
+## Card Review Elements
+
+Understand what the client is asking for.  The client should explain not only the business purpose of the change but any set expectations.
+
+What are the business requirements? Any existing specifications, technical requirements, mock-ups, or other supporting documents we should review?
+
+If the card represents a bug or defect, ask how it can be reproduced as well as the severity of the defect.
+
+Are there target dates either for initial review or going live with the change?  If so, find out more about the nature of those dates.  Are they fixed or flexible?  What's the consequence of missing the date?  Are they vital enough that the scope of the requested work should shift?
+
+Ask questions to clarify anything that you either don't understand or isn't clear in what has been covered so far.
+
+Explore edge case scenarios that may not have been covered to see how the client wants those resolved.
+
+Clarify where in the system these changes should take place.
+
+Identify any related items that may match similar business functionality but may not have been mentioned explicitly by the client.
+
+Bring up any technical concerns with the requested change.  Making sure the client understands the costs and added complexity the card represents is crucial.  They will be making assumptions that things may be easier than they are and as developers we're often best equipped to highlight anything they may not be aware of.
+
+Clarify what exactly completion means for the card.  You will need this to determine when you're done and when the code makes whatever improvement or fix that the client needs.  This information will inform how you test and verify completion.
+
+Ask if the client has any related metrics that can indicate a positive business change in the card.
+
+Before you move on to the next card you should have enough information to do the following:
+
+  -Make a checklist of things you need to do to complete the card
```

Comment:
We'll need to convert this to valid markdown for a bulleted list.  We could make this an ordered list with numerics such as 1., 2.